addons/default/visiosoft/advs-module/src/Adv/Table/AdvTableBuilder.php

```diff
@@ -74,13 +79,18 @@ class AdvTableBuilder extends TableBuilder
         'order_by' => [
             'id' => 'DESC',
         ],
+        'table_view' => 'visiosoft.module.advs::admin/table/table'
     ];
 
     /**
      * The table assets.
      *
      * @var array
      */
-    protected $assets = [];
+    protected $assets = [
+        'scripts.js' => [
+            'visiosoft.module.advs::js/admin/advanced.js'
+        ],
```

Comment:
you only used this javascript for the advanced, why don't you only add it for the advanced type?